splot/_viz_value_by_alpha.py

```diff
@@ -0,0 +1,36 @@
+import geopandas as gpd
+from shapely.geometry import Polygon
+import matplotlib
+
+"""
+Creating Value by Alpha maps
+
+"""
+
+__author__ = ("Stefanie Lumnitz <stefanie.lumitz@gmail.com>")
+
+
+def value_by_alpha(x, alphay, gdf):
+    
+
+
+
+def value_by_alpha(x='attribute', y='alpha_attribute', c, data=geodataframe):
+    rgb = matplotlib.colors.to_rgba_array(c)
+    alpha_channel = x/y.max()
+    rgba = np.vstack((rgb, np.asarray(alpha_channel).reshape(-1,1)))
+    fig, ax = geodataframe.plot(x, color=rgba)
+    return fig, ax
+
+def make_RGBA_array_from_attribute(x):
```

Comment:
This is unfinished, no?

Reply:
sorry, I should have added `value_by_alpha` functionality is dealt with separately in the next pr #28